### DISCUSSION TITLE
Client anti-spam features

### DISCUSSION THREAD

**Discussion by linker3000** [LiveHelperChat/livehelperchat#1826] [General]
Hi,

I have just started to evaluate Live Helper Chat as an option for a customer. Does anyone have feedback on the amount of spam caused by bots starting a chat? Is it a problem?

Anyway, I have been asked: Is there a way to prompt visitors to complete a Recaptcha or simple maths sum to start a chat to restrict the ability for bots to operate the service?

Thanks!

**Comment by remdex** [comment]
Hi,
We have an internal captcha that is not visible by clients, and so far we have not had any problems with thousands of chats.

**Reply by remdex** [reply]
Also there is extensive blocking capabilities by country, IP, cookie etc...

**Reply by linker3000** [reply]
Thanks for the quick response! Also thanks for the great work put into this app.
